ltdconveyor/fastly.py

```diff
@@ -0,0 +1,50 @@
+"""Management of Fastly CDN caching.
+
+See https://docs.fastly.com/api for background.
+"""
+
+import logging
+import requests
+
+from .exceptions import FastlyError
+
+
+logger = logging.getLogger(__name__)
+logger.addHandler(logging.NullHandler())
+
+
+def purge_key(surrogate_key, service_id, api_key):
```

Comment:
I don't see anything wrong with adding a method for this since it is a simple call and avoids an extra dep but it does look like the fastly folks are working on a python sdk.

https://github.com/fastly/fastly-py/blob/master/fastly/fastly.py#L72